tests/conversion/upcastdispatcher.js

```diff
@@ -20,12 +21,15 @@ import ModelWriter from '../../src/model/writer';
 
 import first from '@ckeditor/ckeditor5-utils/src/first';
 import { expectToThrowCKEditorError } from '@ckeditor/ckeditor5-utils/tests/_utils/utils';
+import { StylesProcessor } from '../../src/view/stylesmap';
 
 describe( 'UpcastDispatcher', () => {
-	let model;
+	let model, viewDocument, stylesProcessor;
 
 	beforeEach( () => {
 		model = new Model();
+		stylesProcessor = new StylesProcessor();
+		viewDocument = new ViewDocument( stylesProcessor );
 	} );
```

Comment:
There is no need to expose `stylesProcessor` in tests. It is only used to create `ViewDocument` so 

```js
beforeEach( () => {
    model = new Model();
    viewDocuemnt = new ViewDocument( new StylesProcessor() );
} );
```

should be enough.